data/korath/korath ships.txt

```diff
@@ -211,7 +216,7 @@ ship "Korath Raider"
 	explode "large explosion" 40
 	explode "huge explosion" 10
 	"final explode" "final explosion large"
-	description "The Korath Raider is their warship of choice for plundering neighboring species."
+	description "The Korath Exiles designed the Raider soon after the Dredger. The Raider can be built cheaply in bulk, allowing for non-stop raids of nearby species."
```

Comment:
Not sure I buy this desc if this is actually a Mark 41.  And, I thought I'd throw in a lore bone to the classification...
```suggestion
	description "Ships like the Raider have been the Korath warship of choice for plundering neighbouring species for generations."
	description "This model, like all Exile ships, has been adapted to house as many Korath as possible, as comfortably as possible, inflating its size beyond similarly classed warships. The spaceborne construction techniques employed allow for easy replacement, facilitating continued raids of nearby systems."
```

Reply:
That looks good; although it might be a little all-encompassing. "ships like the Raider" could be a pretty big set. Perhaps it would be better to focus that in a little starting it off with something like "Iterations of the Raider design have been the..." or perhaps "Variations of the Raider design have been the..." That makes it clear that there have been many versions of it over the years, and we're not just talking about a general category of ships.

Reply:
The vagueness was intentional - it could be a big set, it might not be.  However, if specificity is desired here, how's this?  I'm trying to keep it from being wordy:
```suggestion
	description "Successive Raider designs have been the Korath warship of choice for plundering neighbouring species for generations."
	description "This model, like all Exile ships, has been adapted to house as many Korath as possible, as comfortably as possible, inflating its size beyond similarly classed warships. The spaceborne construction techniques employed allow for easy replacement, facilitating continued raids of nearby systems."
```

Reply:
That looks good!